bitmap-goldteam/test/bitmap-constructor-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const expect = require('chai').expect;
+const constructor = require('../lib/bitmap-constructor.js');
+
+describe('File Reading Module', function(err, data) {
+  describe('it should err for a empty buffer', function(err, data) {
+    it('should return an error for empty buffer', function(){
+       // this will check buffer is empty before load
+    });
+  });
+  describe('it should check that read buffer is the same as our buffer object', function(err, data){
+    it('should be the same as our object', function() {
+      // read the buffer and compare to our object
+    })
+  })
+});
```

Comment:
@GLSea1979 @0blu3 @noahgribbin this needs some actual tests - no worries for now, the overall structure of your `describe` and `it` blocks look good - you'll get more comfortable with the actual assertion tests as we continue on in this class